docs/src/app/(public)/(content)/react/components/popover/page.mdx

```diff
@@ -35,5 +37,103 @@ import { Popover } from '@base-ui-components/react/popover';
+
+```jsx title="multiple triggers within root part"
+<Popover.Root>
+  <Popover.Trigger>Trigger 1</Popover.Trigger>
+  <Popover.Trigger>Trigger 2</Popover.Trigger>
+  ...
+</Popover.Root>
+```
+
+```jsx title="multiple detached triggers"
+const demoPopover = Popover.createHandle();
+
+<Popover.Trigger handle={demoPopover}>
+  Trigger 1
+</Popover.Trigger>
+
+<Popover.Trigger handle={demoPopover}>
+  Trigger 2
+</Popover.Trigger>
+
+<Popover.Root handle={demoPopover}>
+  ...
+</Popover.Root>
+```
+
+The popover can render different content depending on which trigger opened it. This is achieved by passing a `payload` to the `Popover.Trigger` and using the function-as-a-child pattern in `Popover.Root`.
+
+The payload can be strongly typed by providing a type argument to the `createHandle` function:
+
+```jsx title="detached triggers with payload"
+const demoPopover = Popover.createHandle<{ text: string }>();
+
+<Popover.Trigger handle={demoPopover} payload={{ text: 'Trigger 1' }}>
+  Trigger 1
+</Popover.Trigger>
+
+<Popover.Trigger handle={demoPopover} payload={{ text: 'Trigger 2' }}>
+  Trigger 2
+</Popover.Trigger>
+
+<Popover.Root handle={demoPopover}>
+  {({ payload }) => (
+    <Popover.Portal>
+      <Popover.Positioner sideOffset={8}>
+        <Popover.Popup className={styles.Popup}>
+          <Popover.Arrow className={styles.Arrow}>
+            <ArrowSvg />
+          </Popover.Arrow>
+          <Popover.Title className={styles.Title}>Popover</Popover.Title>
+          {payload !== undefined && (
+            <Popover.Description className={styles.Description}>
+              This has been opened by {payload.text}
+            </Popover.Description>
+          )}
+        </Popover.Popup>
+      </Popover.Positioner>
+    </Popover.Portal>
+  )}
+</Popover.Root>
+```
+
+Since the popup part is reused between triggers, its position and dimensions can be animated.
+To animate the position, apply CSS transitions to the `left`, `right`, `top`, and `bottom` properties of the **Positioner**.
+Target the **Popup's** `width` and `height` properties to animate dimensions.
+
+The popover also supports transitions between content.
+Wrap the content you wish to transition in the `Viewport` part and apply the transition styles to it.
+The `Viewport` part is only required if you want to animate the content.
+It renders an element (`div` by default) with a `data-activation-direction` attribute set to `left` | `right` | `down` | `up` and a children divs marked with `data-current` (when no transition is active) or `data-next` and `data-previous`.
+The `data-activation-direction` attribute specifies the position of the new trigger relative to the previous one, which can be used to apply different styles based on the direction of the transition.
+
+<Demo path="./demos/detached-triggers-full" />
```

Comment:
This demo's transition styles (timing function, duration, etc.) should ideally closely match Navigation Menu - imo the spring effect looks a bit odd in certain cases.